tools/js-sdk-release-tools/src/hlc/utils/modifyChangelogFileAndBumpVersion.ts

```diff
@@ -17,7 +17,7 @@ export function makeChangesForFirstRelease(packageFolderPath: string, isStableRe
     
 ## ${newVersion} (${date})
 
-The package of ${packageJsonData.name} is using our next generation design principles. To learn more, please refer to our documentation [Quick Start](https://aka.ms/js-track2-quickstart).
+The package of ${packageJsonData.name} is using our next generation design principles. To learn more, please refer to our documentation [Quick Start](https://aka.ms/azsdk/js/mgmt/quickstart ).
```

Comment:
Steps for changelog tools:
- Modify the change under folder `js-sdk-release-tools`
- Build in local version and verify in sdk repo
- Bump the version in sdk-tools repo in packge.json
- Upgrade the lock file also